### PR TITLE
DOCSP-44876-rolling-index-limitation

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Adds rolling index limitation 

## STAGING

https://deploy-preview-457--docs-cluster-to-cluster-sync.netlify.app/reference/limitations/#rolling-index-builds

## JIRA

https://jira.mongodb.org/browse/DOCSP-44876

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -171,3 +171,18 @@ System Collections
 ------------------
 
 .. include:: /includes/collections/behavior-system-collections.rst
+
+Rolling Index Builds
+--------------------
+
+``mongosync`` does not sync source cluster indexes that use a `rolling
+index build
+<https://www.mongodb.com/docs/manual/tutorial/build-indexes-on-replica-sets/>`_.
```
Comment (kennethdyer) [inline source/reference/limitations.txt:180]: The `:ref:` role can find targets on other repos, so we should use that.

If we rename the server file or delete it, this link won't generate an error until the user 404's. A `:ref:` will follow moves and start throwing build errors if we remove the file.

**Review by kennethdyer — CHANGES_REQUESTED** [review]
Thanks @ltran-mdb2, I left a comment for you.

File: source/reference/limitations.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/reference/limitations.txt]: Fixed!

**Review by kennethdyer — APPROVED** [review]
LGTM

**Review by ajayvijayakumar123 — CHANGES_REQUESTED** [review]
Left a suggestion!

File: source/reference/limitations.txt [hunk 2]
```diff
@@ -171,3 +171,17 @@ System Collections
 ------------------
 
 .. include:: /includes/collections/behavior-system-collections.rst
+
+Rolling Index Builds
+--------------------
+
+``mongosync`` does not sync source cluster indexes that use a
```
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/reference/limitations.txt]: Done!

[the remark this answers is not in the document]
Comment (ajayvijayakumar123) [reply source/reference/limitations.txt]: thanks!

**Review by ajayvijayakumar123 — APPROVED** [review]
LGTM!

**Pull request merged**